### PR TITLE
Adding RL Notebooks to the readme file.

### PR DESCRIPTION
**Pull request by ragavvenkatesan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,23 @@ These examples provide quick walkthroughs to get you up and running with Amazon
 - [IP Insights for suspicious logins](introduction_to_amazon_algorithms/ipinsights_login) shows how to train IP Insights on a login events for a web server to identify suspicious login attempts.
 - [Semantic Segmentation](introduction_to_amazon_algorithms/semantic_segmentation_pascalvoc) shows how to train a semantic segmentation algorithm using the Amazon SageMaker Semantic Segmentation algorithm. It also demonstrates how to host the model and produce segmentaion masks and probability of segmentation.
 
+### Amazon SageMaker RL 
+
+The following examples provide quick examples each demonstating different capabilities of Amazon SageMaker RL.
```
Comment (vrkhare) [inline README.md]: "examples" repeated. Perhaps, change with:

The following provide examples demonstating different capabilities of Amazon SageMaker RL.

File: README.md [hunk 2]
```diff
@@ -52,6 +52,23 @@ These examples provide quick walkthroughs to get you up and running with Amazon
 - [IP Insights for suspicious logins](introduction_to_amazon_algorithms/ipinsights_login) shows how to train IP Insights on a login events for a web server to identify suspicious login attempts.
 - [Semantic Segmentation](introduction_to_amazon_algorithms/semantic_segmentation_pascalvoc) shows how to train a semantic segmentation algorithm using the Amazon SageMaker Semantic Segmentation algorithm. It also demonstrates how to host the model and produce segmentaion masks and probability of segmentation.
 
+### Amazon SageMaker RL 
+
+The following examples provide quick examples each demonstating different capabilities of Amazon SageMaker RL. 
+
+- [Carpole using Coach](reinforcement_learning/rl_cartpole_coach) demonstrates the simplest usecase of Amazon SageMaker RL using Intel's coach.
```
Comment (vrkhare) [inline README.md]: Intel's RL Coach

File: README.md [hunk 3]
```diff
@@ -52,6 +52,23 @@ These examples provide quick walkthroughs to get you up and running with Amazon
...
+- [DeepRacer](reinforcement_learning/rl_deepracer_robomaker_coach_gazebo) demonstrates AWS DeepRace trainig using coach in the Gazebo environment.
```
Comment (vrkhare) [inline README.md]: AWS DeepRacer

File: README.md [hunk 4]
```diff
@@ -52,6 +52,23 @@ These examples provide quick walkthroughs to get you up and running with Amazon
...
+- [HVAC using EnergyPlus](reinforcement_learning/rl_hvac_coach_energyplus) demonstrates the training of HVAC systems using the EnergyPlus environment.
+- [Knapsack Problem](reinforcement_learning/rl_knapsack_coach_custom) demonstrates how to solve the knapsack problem using a custom environment. 
+- [Mountain Car](reinforcement_learning/rl_mountain_car_coach_gymEnv) Mountain car is a classic RL problem. This notebook explains how to solve this using the OpenAI Gym environment. 
+- [Distributed Neural Network Compression](reinforcement_learning/rl_network_compression_ray_custom) This notebook explains how to compress ResNets using RL, using a custom environment and the Ray toolkit.
```
Comment (vrkhare) [inline README.md]: RLlib toolkit

File: README.md [hunk 5]
```diff
@@ -52,6 +52,23 @@ These examples provide quick walkthroughs to get you up and running with Amazon
...
+- [Turtlebot Tracker](reinforcement_learning/rl_objecttracker_robomaker_coach_gazebo) This notebook demonstrates object tracking using Robomaker and Coach in the Gazebo environment.
```
Comment (vrkhare) [inline README.md]: AWS RoboMaker and RL Coach

File: README.md [hunk 6]
```diff
@@ -52,6 +52,23 @@ These examples provide quick walkthroughs to get you up and running with Amazon
...
+- [Portfolio Management](reinforcement_learning/rl_portfolio_management_coach_customEnv) This notebook uses a custom gym environment to manage multiple financial investments.
```
Comment (vrkhare) [inline README.md]: Gym

File: README.md [hunk 7]
```diff
@@ -52,6 +52,23 @@ These examples provide quick walkthroughs to get you up and running with Amazon
...
+- [Autoscaling](reinforcement_learning/rl_predictive_autoscaling_coach_customEnv) demonstrates how to adjust load depending on demand. This uses coach and a custom environment.
```
Comment (vrkhare) [inline README.md]: how to auto-scale ... RL Coach

File: README.md [hunk 8]
```diff
@@ -52,6 +52,23 @@ These examples provide quick walkthroughs to get you up and running with Amazon
...
+- [Roboschool](reinforcement_learning/rl_roboschool_ray) is an open source physics simulator that is commonly used to train RL policies for robotic systems. This notebook demonstrates training a few agents using it.
+- [Stable Baselines](reinforcement_learning/rl_roboschool_stable_baselines) In this notebook example, we will make the HalfCheetah agent learn to walk using the stable-baselines a set of improved implementations of Reinforcement Learning (RL) algorithms based on OpenAI Baselines.
```
Comment (vrkhare) [inline README.md]: rephrase - "using the stable-baselines a set of improved implementations of Reinforcement Learning (RL) algorithms based on OpenAI Baselines."

File: README.md (hunk 4 above)
Comment (ragavvenkatesan) [reply README.md]: Done.

File: README.md (hunk 6 above)
Comment (ragavvenkatesan) [reply README.md]: Done.

File: README.md (hunk 8 above)
Comment (ragavvenkatesan) [reply README.md]: Done.

File: README.md (hunk 5 above)
Comment (ragavvenkatesan) [reply README.md]: Done.

**Pull request merged**